distribution/server-quarkus/src/main/README.txt

```diff
@@ -0,0 +1,12 @@
+Keycloak server powered by Quarkus.
```

Comment:
Change "Keycloak server powered by Quarkus" to "Keycloak.X"

Reply:
Changed as requested